### DISCUSSION TITLE
I made a video about my bot. And it's source code looks like a pawn

### DISCUSSION THREAD

**Discussion by GameTechExplained** [SebLague/Chess-Challenge#493] [Show and tell]
I hope it is okay to share this here. I made a short ten minute video about the process of writing my bot on my teeny tiny 17 subscriber YouTube channel. Feel free to check it out while waiting for Sebastian's big resolution: https://youtu.be/5vsLmM756LA

**Comment by LiteApplication** [comment]
I saw your video, liked it a lot :)
